regression/cbmc-java/NondetInit/test_lazy.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+CORE symex-driven-lazy-loading-expected-failure
```

Comment:
Probably good to add a comment to this `test.desc` explaining why this is an expected failure

Reply:
I see it is because of the `lazy-methods` flag. I still think they'd benefit from a uniform comment, but you may disagree.

Reply:
+1 for having a comment on each of the skipped desc to mention the reason they are skipped.